### PR TITLE
Remove old acl create node refactor file service

### PR DESCRIPTION
**Pull request by frankyma**
Refactor the 3 file creates with the new Create pattern, then remove the db.createNode and its helpers

### CODE REVIEW

File: src/components/file/file.repository.ts [hunk 1]
```diff
@@ -194,9 +196,11 @@ export class FileRepository {
         .with('*')
         .optionalMatch([
           node('requestingUser'),
-          relation('in', '', 'member'),
-          node('acl', 'ACL', { [camelCase(`canRead-${prop}`)]: true }),
-          relation('out', '', 'toNode'),
+          relation('in', '', 'member*1..'),
```
Comment (CarsonF) [inline src/components/file/file.repository.ts:199]: @michaelmarshall just dropped this with #1193 right?
```suggestion
          relation('in', '', 'member'),
```

Comment (frankyma) [reply src/components/file/file.repository.ts]: removed

File: src/components/file/file.repository.ts [hunk 2]
```diff
@@ -255,97 +259,167 @@ export class FileRepository {
...
+        undefined,
+        input.id
+      )
+      .return('node.id as id')
+      .asResult<{ id: string }>();
+
+    const result = await createFile.first();
+
+    if (!result) {
+      throw new ServerException('failed to create a FileVersion');
```
Comment (CarsonF) [inline src/components/file/file.repository.ts:421]:
```suggestion
      throw new ServerException('Failed to create file version');
```

File: src/components/file/file.repository.ts [hunk 3]
```diff
@@ -255,97 +259,167 @@ export class FileRepository {
...
+      .call(createBaseNode, ['File', 'FileNode'], props, {
+        owningOrgId: session.owningOrgId,
+      })
+      .return('node.id as id')
+      .asResult<{ id: string }>();
+
+    const result = await createFile.first();
+
+    if (!result) {
+      throw new ServerException('failed to create a File');
```
Comment (CarsonF) [inline src/components/file/file.repository.ts:339]:
```suggestion
      throw new ServerException('Failed to create file');
```

File: src/components/file/file.repository.ts [hunk 4]
```diff
@@ -255,97 +259,167 @@ export class FileRepository {
...
+      .call(createBaseNode, ['Directory', 'FileNode'], props, {
+        owningOrgId: session.owningOrgId,
+      })
+      .return('node.id as id')
+      .asResult<{ id: string }>();
+
+    const result = await createFile.first();
+
+    if (!result) {
+      throw new ServerException('failed to create a Directory');
```
Comment (CarsonF) [inline src/components/file/file.repository.ts:292]:
```suggestion
      throw new ServerException('Failed to create directory');
```

File: src/components/file/file.repository.ts (hunk 2 above)
Comment (frankyma) [reply src/components/file/file.repository.ts]: changed

File: src/components/file/file.repository.ts (hunk 4 above)
Comment (frankyma) [reply src/components/file/file.repository.ts]: changed

File: src/components/file/file.repository.ts (hunk 3 above)
Comment (frankyma) [reply src/components/file/file.repository.ts]: changed

**Review by CarsonF — APPROVED** [review]
Thanks @frankyma!

**Pull request merged**